### ISSUE TITLE
SVN STATUS ran without me enabling

### ISSUE THREAD

**Issue by kcgp007** [romkatv/powerlevel10k#2412]
The default settings are used in the `.p10k.zsh` file
`typeset -g POWERLEVEL9K_VCS_BACKENDS=(git)`
But every time I enter the SVN project folder, I will get stuck, and then I can see the running process of `svn status` in htop, and the tty will not enter the folder until the process ends.
In other places, I use the default configuration without any changes. I don’t know if there are any other configurations that need to be modified to take effect.
In addition, I am using Arch Linux, maybe there is something different in the default configuration

**Comment by romkatv** [comment]
What happens if you disable powerlevel10k or replace it with another theme?

**Comment by kcgp007** [comment]
I tried to disable the powerlevel10k theme in zsh, and also commented the reference of .p10k.zsh, so that there will be no stuck problem after modification

**Comment by romkatv** [comment]
Are you saying that after disabling powerlevel10k the problem has disappeared? If so, the issue must be caused by the powerlevel10k configuration that you are using. Try the following:

1. Enable powerlevel10k.
2. Add the following at the very bottom of `~/.zshrc`:
   ```zsh
   unset -m 'POWERLEVEL9K_*'
   POWERLEVEL9K_VCS_BACKENDS=(git)
   ```

Does this help?

**Comment by kcgp007** [comment]
The result is still the same, "svn status" appears, and then stuck

![image](https://github.com/romkatv/powerlevel10k/assets/13428233/78b36c9b-cb46-4351-94c3-0f2d2b0d6695)

**Comment by romkatv** [comment]
I can see on your screenshot that something went wrong. Have you follow the instructions I posted? Have you restarted your terminal afterwards?

**Comment by kcgp007** [comment]
Yes, I configured and restarted, I will send out my complete configuration

```shell
# plugin
# zsh-autosuggestions
source /usr/share/zsh/plugins/zsh-autosuggestions/zsh-autosuggestions.zsh
# zsh-syntax-highlighting
source /usr/share/zsh/plugins/zsh-syntax-highlighting/zsh-syntax-highlighting.zsh
# pkgfile
source /usr/share/doc/pkgfile/command-not-found.zsh
# zoxide
eval "$(zoxide init zsh)"
# zsh-fzf-history-search
source /usr/share/zsh/plugins/zsh-fzf-history-search/zsh-fzf-history-search.plugin.zsh
# zsh-thefuck-git
source /usr/share/zsh/plugins/zsh-thefuck-git/zsh-thefuck.plugin.zsh
# zsh-extract-git
source /usr/share/zsh/plugins/zsh-extract/extract.plugin.zsh
# fzf-tab-git
source /usr/share/zsh/plugins/fzf-tab-git/fzf-tab.plugin.zsh
# set list-colors to enable filename colorizing
zstyle ':completion:*' list-colors ${(s.:.)LS_COLORS}
# zsh-theme-powerlevel10k
source /usr/share/zsh-theme-powerlevel10k/powerlevel10k.zsh-theme

unset -m 'POWERLEVEL9K_*'
POWERLEVEL9K_VCS_BACKENDS=(git)

setopt HIST_IGNORE_DUPS
setopt HIST_IGNORE_ALL_DUPS
setopt HIST_REDUCE_BLANKS

# exa
alias exa="exa --icons --git"
alias ls="exa"
alias ll="exa -l"
alias la="exa -la"
alias li="exa -l --git-ignore"
# advcpmv
alias cp="cpg -g"
alias mv="mvg -g"

# To customize prompt, run `p10k configure` or edit ~/.p10k.zsh.
[[ ! -f ~/.p10k.zsh ]] || source ~/.p10k.zsh
```

Another point, I don't know if it will help to judge the cause of the error. When I press Ctrl+C when I am stuck, an error "svn: E200015: Caught signal" will appear, and exit the theme interface of powerlevel10k, changing to Return to zsh's default theme

**Comment by romkatv** [comment]
> ```shell
> unset -m 'POWERLEVEL9K_*'
> POWERLEVEL9K_VCS_BACKENDS=(git)
> ```

You need to move this block **to the very bottom** of `.zshrc`.

**Comment by kcgp007** [comment]
Still appeared. .
![image](https://github.com/romkatv/powerlevel10k/assets/13428233/7df31136-73ea-44f0-a0af-5be14ea43370)

**Comment by romkatv** [comment]
Try this:

1. Run this:
   ```zsh
   zsh -f
   ```
3. Then run this:
   ```zsh
   POWERLEVEL9K_VCS_BACKENDS=(git)
   source /usr/share/zsh-theme-powerlevel10k/powerlevel10k.zsh-theme
   ```

Does this help?

**Comment by kcgp007** [comment]
This is ok, is it because of my other configuration or plug-in problems?

**Comment by romkatv** [comment]
Yes.

**Comment by kcgp007** [comment]
Oh, yes, thank you. After your prompt, I found the reason. It is the configuration in grml-zsh-config. For some reason, it triggers the SVN STATUS check in powerlevel10k

**Comment by romkatv** [comment]
I don't think grml-zsh-config knows about powerlevel10k. It's probably invoking `svn status` all the time regardless of whether you are using powerlevel10k or not. In any case, powerlevel10k does not invoke `svn status`.
